ncclient/manager.py

```diff
@@ -167,178 +155,3 @@ def connect(*args, **kwds):
         else:
             return connect_ssh(*args, **kwds)
 
-
-class Manager(object):
```

Comment:
I don't see the need to create a new `sync_manager.py` file. This makes it really hard to track any diffs in the original `class Manager`. Introducing the new `async_manager.py` is fair enough.